aii-ks/src/main/perl/ks.pm

```diff
@@ -17,7 +43,7 @@ our $EC = LC::Exception::Context->new->will_store_all;
 
 our $this_app = $main::this_app;
 # Modules that may be interesting for hooks.
-our @EXPORT_OK = qw (ksuserhooks ksinstall_rpm get_repos replace_repo_glob);
+our @EXPORT_OK = qw (ksuserhooks ksinstall_rpm);
```

Comment:
why is this being removed? @jouvin is your PR based on latest master?

Reply:
You are right, it based on 21.12.1, I'll fix this.